storage/log_iterator.go

```diff
@@ -26,8 +27,8 @@ type LogIterator interface {
 	// IsReversed returns true if the iterator is in reverse order and
 	// false otherwise.
 	IsReversed() bool
-	// Channel returns a channel to receive the iterator's logs.
-	Channel(context.Context) chan *model.LogLineItem
+	// Stream returns a chan of log lines from the iterator.
+	Stream(context.Context) chan *model.LogLineItem
```

Comment:
Trying to avoid Hungarian notation-y things, [this](https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat) is a good article on it.